package.json

```diff
@@ -39,6 +39,7 @@
   "devDependencies": {
     "@babel/core": "^7.6.4",
     "@jest/core": "26.6.3",
+    "@react-three/fiber": "^7.0.26",
```

Comment:
was the three / r3f package upgrade deferred?

Reply:
Nope, that's a separate effort, this is related just merging what we have in our private fork to the public repo.